docs/source/api_reference/pipeline.rst

```diff
@@ -0,0 +1,17 @@
+.. _regression_ref:
+
+Graphical Pipeline
```

Comment:
I think this should also list `make_pipeline`, so there's two objects here - the linear and the graphical one.